libraries/rush-lib/src/logic/PackageJsonUpdater.ts

```diff
@@ -78,6 +83,27 @@ export interface IPackageJsonUpdaterRushAddOptions {
   variant?: string | undefined;
 }
 
+/**
+ * Options for remove a dependency to a particular project.
+ */
+export interface IPackageJsonUpdaterRushRemoveOptions {
```

Comment:
This and `IPackageJsonUpdaterRushAddOptions` should be able to extend from a common base.

Reply:
done